README.md

```diff
@@ -89,6 +89,26 @@ module MyCoolGem
 end
 ```
 
+#### Customize env variable names prefix
+
+By default, Anyway Config will use config name with stripped underscores as a prefix for env variable names (e.g.
+`config_name :my_app` will result to parsing `MYAPP_HOST` variable, not `MY_APP_HOST`). You can set env prefix
+explicitly, and it will be used as is:
+
+```ruby
+module MyCoolGem
+  class Config < Anyway::Config
+    config_name :cool_gem
+    env_prefix :really_cool # now variables, starting wih `REALLY_COOL_`, will be parsed
+    attr_config user: 'root', password: 'root', host: 'localhost', options: {}
+  end
+end
+```
+
+**DEPRECATION WARNING** In the 1.3 version no stripping will be applied on config_names by default, so if you use explicit config names with
```

Comment:
The next version is 1.3, where we introduce this warning. Thus the change will appear in 1.4.0.

Reply:
Ah, ok, I thought, it will be released in the next patch version :)